examples/psp22_extensions/burnable/lib.rs

```diff
@@ -4,17 +4,36 @@
 #[brush::contract]
 pub mod my_psp22_burnable {
     use brush::contracts::psp22::extensions::burnable::*;
-    use ink_prelude::vec::Vec;
+    use ink_prelude::{
+        vec::Vec,
+        string::String
+    };
 
     #[ink(storage)]
     #[derive(Default, PSP22Storage)]
     pub struct MyPSP22 {
         #[PSP22StorageField]
         psp22: PSP22Data,
+        // fields for hater logic
+        hated_account: AccountId,
```

Comment:
You don't need to test `before_token_transfer` and `after_token_transfer` in integration tests. So you don't need to modify example and integration test to test that=)